dvc/cli.py

```diff
@@ -117,6 +120,7 @@ def parse_args(argv=None):
                         help='Lock data item - disable reproduction.')
     run_parser.add_argument('-f',
                         '--file',
+                        default=Stage.STAGE_FILE,
```

Comment:
We should not, as it will require us to convert all paths in cmd to be relative to that dir, which is bad as there might be non obvious cases. I suggest we stick to STAGE_FILE for now.